e2e/windows/e2e-scenario.sh

```diff
@@ -43,7 +43,8 @@ collect-logs() {
     set -x
 }
 
-RESOURCE_GROUP_NAME="$RESOURCE_GROUP_NAME"-"$WINDOWS_E2E_IMAGE"-v2
+K8S_VERSION=$(echo $KUBERNETES_VERSION | tr '.' '-')
+RESOURCE_GROUP_NAME="$RESOURCE_GROUP_NAME"-"$WINDOWS_E2E_IMAGE"-"$K8S_VERSION"
 
 DEPLOYMENT_VMSS_NAME="$(mktemp -u winXXXXX | tr '[:upper:]' '[:lower:]')"
```

Comment:
this is for @junjiezhang1997 : can we use a meaningful name `winXXXXX `?

Reply:
Do you mean using a meaningful name for DEPLOYMENT_VMSS_NAME? We randomly generate the DEPLOYMENT_VMSS_NAME to avoid duplicate names between different pipelines.